client/sites-dashboard/components/sites-container.tsx

```diff
@@ -0,0 +1,139 @@
+import { ClassNames } from '@emotion/react';
+import styled from '@emotion/styled';
+import { createInterpolateElement } from '@wordpress/element';
+import { useI18n } from '@wordpress/react-i18n';
+import EmptyContent from 'calypso/components/empty-content';
+import { SiteData } from 'calypso/state/ui/selectors/site-data';
+import { SitesTable } from './sites-table';
+
+const EmptyContainer = styled.div`
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	height: 600px;
```

Comment:
The problem with this `height` property is that it might overflow on smaller, mobile screens. Setting `100%` won't work because the browser won't be able to calculate the remaining space, and `100vh` won't work as well because there are other elements taking part of the height, so we won't be able to make this content centered easily.

I suggest removing the property. The results aren't bad either and we will respect the user's viewport.

Reply:
yeah I was wondering how to get it centered otherwise. we probably would have to use top margin instead and transform by 50% of viewport height?